optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -479,7 +482,7 @@ def best_params(self) -> Dict[str, Any]:
             params.update(self.lgbm_params)
             return params
 
-    def get_best_booster(self) -> "lgb.Booster":
+    def get_best_booster(self) -> "lgb.Booster" | "lgb.CVBooster":
```

Comment:
[Question] Does this method returns `lgb.CVBooster` instances? Or, do you add `lgb.CVBooster` class only to fix the CI?

Reply:
In my understanding, this method does not return `lgb.CVBooster` when called from subclass `LightGBMTuner`. However, `get_best_booster` is overwritten in subclass `LightGBMTunerCV` and without this change causes mypy error. 
Shall I move the method from `_LightGBMBaseTuner` to `LightGBMTuner`?

Reply:
Sure. I fixed it.